user-docs/latest/getting-started/introduction.md

```diff
@@ -1,3 +1,68 @@
+
+- **Versions** <br> Strapi is constantly evolving and growing. This implies that new releases are quite frequent, to improve what is already available but also to add new features to Strapi. For every new Strapi version, we communicate through our main channels and by sending notifications both on your terminal (when launching your Strapi application), and on your application's admin panel. We always recommend to use the latest version, especially since our documentation, including this user guide, are only aligned with the latest version. If you are on an older version of Strapi, some information in this documentation may not apply to your application. 
+
+- **Roles and Permissions** <br> Some features of the admin panel, as well as the content created with Strapi itself, are ruled by a system of permissions. From your Strapi application, you have the possibility to define, at a detailed level, the roles and permissions of all admin panel users and end-users. In this user guide, all features and possible options are documented. It is however possible, depending on your role and permissions, that you may not be able to access all these features and options. In that case, please refer to the main Super Admin of your Strapi application.
+
+With all this in mind, you should be ready to start your Strapi experience!
+
+
+## Accessing the admin panel
+
+The admin panel is the back office of your Strapi application. From the admin panel, you will be able to manage content types, and write their actual content. It is also from the admin panel that you will manage users, both admin panel users and end-users of your Strapi application.
+
+::: tip NOTE
+Strapi can either be used locally, or hosted on a server, making it available at a specific URL. No matter how you choose to use Strapi, your application will work the same way. The main difference is that, if using it locally, you will need to use your terminal to launch Strapi.
+:::
+
+![User profile](../assets/getting-started/login-page.png)
+
+To access the admin panel:
+
+1. Launch Strapi.
+2. Click on the **Open the administration** button.
+3. Enter your credentials to log in.
+4. Click on the **Log in** button. You should be redirected to the homepage of the admin panel.
+
+::: tip 💡 TIP
+Since the link which where to be redirected after clicking on **Log in** is customizable, it is possible to not be redirected to the homepage of the admin panel. To find that homepage, you can add `/admin` at the end of your Strapi application's URL.
+:::
+
+
+## Setting up your user profile
+
+If you are a new user, we recommend making sure your user profile is all set, before diving into your Strapi application. From your admin panel user profile, you are able to modify your user information, such as name, username, email or password.
```

Comment:
```diff
- If you are a new user
+ If you are a new admin panel user
OR
+ If you are a new Strapi user
```

I don't know here, related to the Admin VS End users topic and depending on the meaning of the sentence.
I'm not sure there.

Reply:
I think it's meant in the context of the admin panel user profile, so the former suggestion.